token/program-2022/src/extension/transfer_fee/mod.rs

```diff
@@ -351,7 +362,7 @@ pub(crate) mod test {
             transfer_fee.calculate_inverse_fee(one).unwrap()
         );
         // zero is 1 token
-        assert_eq!(minimum_fee, transfer_fee.calculate_inverse_fee(0).unwrap());
+        // assert_eq!(minimum_fee, transfer_fee.calculate_inverse_fee(0).unwrap());
```

Comment:
This should still work too

Reply:
Yep, this is related to the comment above. Assume that bp is positive. If post fee amount is 0, then `calculate_pre_fee_amount` could produce either 0 or 1, which results in fee of either 0 or 1.

Currently, `calculate_pre_fee_amount` produces pre fee of 0 on post fee 0, so this test should fail since `minimum_fee` is 1. If the current behavior of `calculate_pre_fee_amount` is fine, then `minimum_fee` should be replaced by 0 in this test.

Reply:
Per our discussion, let's have this be 0 then!